nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/_app.component-theme.scss

```diff
@@ -26,14 +26,12 @@
     // Get the color palette from the color-config.
     $primary-palette: map.get($color-config, 'primary');
     $accent-palette: map.get($color-config, 'accent');
-    $warn-palette: map.get($color-config, 'warn');
     $canvas-primary-palette: map.get($canvas-color-config, 'primary');
 
     // Get hues from palette
     $primary-palette-300: mat.get-color-from-palette($primary-palette, 300);
     $primary-palette-500: mat.get-color-from-palette($primary-palette, 500);
     $accent-palette-A400: mat.get-color-from-palette($accent-palette, 'A400');
-    $warn-palette-500: mat.get-color-from-palette($warn-palette, 500);
```

Comment:
Same thing. This should have been removed in the snackbar theming PR. I cleaned them up here.